scripts/generate-releases.sh

```diff
@@ -2,64 +2,84 @@
 
 set -euo pipefail
 
-if [[ "$#" -lt 1 || "$#" -gt 3 ]]; then
-    echo "USAGE: ./generate-releases.sh <ENVIRONMENT> [COMMIT] [BRANCH]" >&2
-    echo "" >&2
-    echo "ENVIRONMENT - allowed values: staging|prod" >&2
-    echo "COMMIT - a 40 character-long SHA of the commit to pull Snapshots only with this commit label for the Release. Default: currently checked out commit" >&2
-    echo "BRANCH - an optional parameter to specify git branch name for filtering snapshots by having branch name in annotations. Default: currently checked out branch" >&2
-    echo "" >&2
-    echo "You must have your KUBECONFIG point to the Konflux cluster, see https://spaces.redhat.com/pages/viewpage.action?pageId=407312060#HowtoeverythingKonfluxforRHACS-GettingocCLItoworkwithKonflux." >&2
-    exit 1
-fi
-
-ENVIRONMENT="$1"
-COMMIT="${2:-$(git rev-parse HEAD)}"
-BRANCH="${3:-$(git rev-parse --abbrev-ref HEAD)}"
-
-release_name="${ENVIRONMENT}-$(git rev-parse --short HEAD)-$(date +'%Y-%m-%d-%H-%M')"
-# Fetch the list of snapshots for COMMIT and BRANCH. 
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+source "$SCRIPT_DIR/helpers.sh"
+
+# Describe the usage of the script if not enough arguments are provided.
+usage() {
+    if [[ "$#" -lt 1 || "$#" -gt 3 ]]; then
```

Comment:
Optional suggestion:
```suggestion
    if [[ "$#" -lt 1 || "$#" -gt 3 || "${1:-}" == "--help" ]]; then
```
I did not test it though. Up to you.

Reply:
It works, I like it